index.html

```diff
@@ -16,30 +16,38 @@
     <header>
       <div class="header-cont">
         <div class="nav-cont">
+          <ul class="nav-cont-ul">
+            <li class="nav-cont-li"><a href="">Hello</a></li>
+            <li class="nav-cont-li"><a href="">Portfolio</a></li>            
+            <li class="nav-cont-li"><a href="">About</a></li>            
+            <li class="nav-cont-li"><a href="">Contact</a></li>
+          </ul>
           <nav class="nav-bar">
             <i class="fa-solid fa-bars fa-xl"></i>
           </nav>
         </div>
-        <div>
-          <h1 class="header-h1">
-            Fuad <br> Nabiyev
-          </h1>
-          <p class="header-p">
-            Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.
-            If you like what you see and have a project you need coded, don’t hesitate and contact me.
-          </p>
+        <div class="header-texts">
+          <div class="header-text">
+            <h1 class="header-h1">
+              Fuad <br> Nabiyev
+            </h1>
+            <p class="header-p">
+              Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.<br>
+              If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+          </div>
+          <button class="start-btn">
+              Start collaboration
+          </button>
         </div>
-        <button class="start-btn">
-            Start collaboration
-        </button>
       </div>
```

Comment:
- [ ] Kindly make sure the design conforms with that of the figma design at all screen sizes of the desktop version starting from screen width 768px. There is also a blank space on the right side which is not suppose to be there. Kindly check and make sure it conforms to the figma design. :pray: 

Your header             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-06 12-55-07](https://user-images.githubusercontent.com/65233860/194319920-b49e5eed-f255-445b-9f9e-87e8c79e9d55.png) |  ![Screenshot from 2022-10-06 13-02-58](https://user-images.githubusercontent.com/65233860/194320285-41de3873-2daf-49fa-9843-757d022eab77.png)

The same applies to all the other sections including but not limited to the `project` and about `section`

Your about section             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-06 12-55-46](https://user-images.githubusercontent.com/65233860/194321345-9285d54c-56c7-4fe8-ad9d-5f72316f1fa9.png)  |  ![Screenshot from 2022-10-06 13-03-25](https://user-images.githubusercontent.com/65233860/194321390-3ae6c1fd-e5a1-4800-93df-8bbb51f7d0af.png)


Your contact section             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-06 12-56-03](https://user-images.githubusercontent.com/65233860/194322092-36da1c06-4e3d-456e-83a3-39708869d8fd.png)  |  ![Screenshot from 2022-10-06 13-03-33](https://user-images.githubusercontent.com/65233860/194321630-811bef9d-d639-48bb-bd9e-ced72d86ac9d.png)